### PR TITLE
Replaced ActionableMessage component in srp-input

### PR DESCRIPTION
**Pull request by pritam1813**
## Explanation

- This PR replaces all `ActionableMessage` components in `ui/components/app/srp-input/srp-input.js` with `BannerAlert` component .
- This PR is a part of #19528 

### Before

[srp-input before](https://github.com/MetaMask/metamask-extension/assets/46365255/96b01300-9b41-4587-b68f-deb6900e7550)

### After

[srp-input-after](https://github.com/MetaMask/metamask-extension/assets/46365255/a3b6b93d-cfdd-4693-aa22-df4082d5eb91)

## Manual Testing Steps

- Check the Storybook build of `srp-input` component
- Create new development build 
## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by pritam1813** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by pritam1813** [comment]
@georgewrmarshall

File: ui/components/app/srp-input/srp-input.js [hunk 1]
```diff
@@ -132,11 +131,10 @@ export default function SrpInput({ onChange, srpText }) {
           {srpText}
         </Text>
       </label>
-      <ActionableMessage
+      <BannerAlert
         className="import-srp__paste-tip"
-        iconFillColor="var(--color-info-default)"
-        message={t('srpPasteTip')}
-        useIcon
+        severity="info"
```
Comment (georgewrmarshall) [inline ui/components/app/srp-input/srp-input.js:136]: Could we use the `Severity` enum from `ui/helpers/constants/design-system.ts` for the severities?
```
import { Severity } from '../../../helpers/constants/design-system';
```

```suggestion
        severity={Severity.Info}
```

File: ui/components/app/srp-input/srp-input.js [hunk 2]
```diff
@@ -200,25 +198,19 @@ export default function SrpInput({ onChange, srpText }) {
...
       </div>
       {srpError ? (
-        <ActionableMessage
+        <BannerAlert
           className="import-srp__srp-error"
-          iconFillColor="var(--color-error-default)"
-          message={srpError}
-          type="danger"
-          useIcon
+          severity="danger"
```
Comment (georgewrmarshall) [inline ui/components/app/srp-input/srp-input.js:203]: Same here
```suggestion
          severity={Severity.Danger}
```

File: ui/components/app/srp-input/srp-input.js [hunk 3]
```diff
@@ -200,25 +198,19 @@ export default function SrpInput({ onChange, srpText }) {
...
           className="import-srp__srp-too-many-words-error"
-          iconFillColor="var(--color-error-default)"
-          message={t('srpPasteFailedTooManyWords')}
-          primaryAction={{
-            label: t('dismiss'),
-            onClick: () => setPasteFailed(false),
-          }}
-          type="danger"
-          useIcon
+          severity="danger"
```
Comment (georgewrmarshall) [inline ui/components/app/srp-input/srp-input.js:210]: Same here
```suggestion
          severity={Severity.Danger}
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hey @pritam1813, this is looking good! I have a couple of suggestions and it looks like there are some failing e2e tests that need to be resolved. The way I approach debugging these is to search for the test name and then the CSS selector that the test is looking for. I think we could update this to a test id if possible

Screenshot 2023-07-24 at 9 00 11 AM

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Thanks for your contribution @pritam1813

File: ui/components/app/srp-input/srp-input.js (hunk 1 above)
Comment (georgewrmarshall) [reply ui/components/app/srp-input/srp-input.js]: I think that makes sense!

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Ah apologies still need to update e2e tests

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Excellent work @pritam1813 👍 Really nice PR description
- checkout branch and checked storybook for all 3 banners

**Comment by georgewrmarshall** [comment]
Hey @pritam1813, looks like you may have to update some e2e tests here I would suggest adding data-testids to the banners and using those in the tests. Thanks

File: test/e2e/helpers.js [hunk 4]
```diff
@@ -426,7 +426,7 @@ const importWrongSRPOnboardingFlow = async (driver, seedPhrase) => {
   );
 
   const warningText = 'Invalid Secret Recovery Phrase';
-  const warnings = await driver.findElements('.actionable-message__message');
+  const warnings = await driver.findElements('.banner-alert-text');
```
Comment (georgewrmarshall) [inline test/e2e/helpers.js:429]: To make this more specific to the import srp page could we prefix this className?
```suggestion
  const warnings = await driver.findElements('.import-srp__banner-alert-text');
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looking good! Once suggestion to increase specificity

File: test/e2e/helpers.js (hunk 4 above)
Comment (pritam1813) [reply test/e2e/helpers.js]: Done. Good Suggestion.

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Thanks for your contribution @pritam1813

**Review by garrettbear — APPROVED** [review]

**Pull request merged**